test/test_auth_api.js

```diff
@@ -119,11 +119,12 @@ describe('Authentication API', () => {
     it('should respond 400 if the client has no redirect url', function*() {
       const adminToken = yield loginAsAdmin(server);
       const client = yield createClient(server, adminToken, { name: 'test' });
-      const clientToken = yield loginAsClient(server, client);
+      const authToken = yield signClientRequest(client, {
+        redirectUrl: redirectUrls[0]
+      });
```

Comment:
nit: I would indent like this:
```javascript
const authToken = yield signClientRequest(
  client, { redirectUrl: redirectUrls[0] }
);
```
I find it slightly more readable to have the 2 parameters in the same line :)